code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1514,6 +1514,25 @@
 
 	M.adjustToxLoss(1)
 
+/datum/reagent/vomit
```

Comment:
Does this need to exist? Vomit isn't really a substance, it's a mixture of stuff we already have i.e. nutriment and hydrochloric acid.